ui/src/client/Clients.tsx

```diff
@@ -119,6 +123,17 @@ const Row: SFC<IRowProps> = ({name, value, fEdit, fDelete}) => (
                 style={{display: 'flex', alignItems: 'center', width: 200}}
             />
         </TableCell>
+        <TableCell>
```

Comment:
The token overlaps the last used column, we could you make this non overlapping?

![image](https://github.com/gotify/server/assets/14895212/5e0ca290-c272-4afa-b646-94160df7daaa)